components/feature/addons/src/main/java/mozilla/components/feature/addons/update/AddonUpdater.kt

```diff
@@ -78,6 +79,18 @@ interface AddonUpdater {
         onPermissionsGranted: ((Boolean) -> Unit)
     )
 
+    /**
+     * Registers the given [extensions] for periodically check for new updates.
```

Comment:
nit: Registers the given [extensions] for periodic updates.

Reply:
👍